R/rules-replacement.R

```diff
@@ -0,0 +1,14 @@
+force_assignment_op <- function(pd) {
+  to_replace <- pd$token == "EQ_ASSIGN"
```

Comment:
Is this replacement safe in all circumstances? I'd like to differentiate between styling options that only add/remove whitespace (because there we can easily prove that the code is unchanged) and those that do change the code. For a new package, the user can then gradually turn on styling, and review potentially breaking changes separately.